README.md

```diff
@@ -1,10 +1,54 @@
 Protractor helper's library
 ===============================
 
+[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper)
+
 This library contains a file (`index.js`) with different helper methods that can be used together with Protractor for creating robust end-to-end tests.
 
 Many of the helper methods on this library uses `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps on avoiding trying to interact with elements when they are still not ready for it, which helps avoiding test flakiness.
 
+## Summary
+
+- [Basic example](#basic-example)
+  - [Example without using the protractor-helper module](#example-without-using-the-protractor-helper-module)
+  - [Example using the protractor-helper module](#example-using-the-protractor-helper-module)
+- [Installation](#installation)
+- [How to use and examples](#how-to-use-and-examples)
+- [Available helpers](#available-helpers)
+  <details><p><summary>Open to see all available helpers</summary>
+
+  - [`setTimeout`](#settimeout)
+  - [`getBodyElementFromCurrentBrowserOrBrowserInstance`](#getbodyelementfromcurrentbrowserorbrowserinstance)
+  - [`openNewBrowserInTheSamePage`](#opennewbrowserinthesamepage)
+  - [`isCurrentUrlDifferentFromBaseUrl`](#iscurrenturldifferentfrombaseurl)
+  - [`waitForElementPresence`](#waitforelementpresence)
+  - [`waitForElementNotToBePresent`](#waitforelementnottobepresent)
+  - [`waitForElementVisibility`](#waitforelementvisibility)
+  - [`waitForElementNotToBeVisible`](#waitforelementnottobevisible)
+  - [`clickWhenClickable`](#clickwhenclickable)
+  - [`fillFieldWithTextWhenVisible`](#fillfieldwithtextwhenvisible)
+  - [`fillInputFieldWithFileWhenPresent`](#fillinputfieldwithfilewhenpresent)
+  - [`clearFieldWhenVisible`](#clearfieldwhenvisible)
+  - [`clearFieldWhenVisibleAndFillItWithText`](#clearfieldwhenvisibleandfillitwithtext)
+  - [`tapWhenTappable`](#tapwhentappable)
+  - [`waitForTextToBePresentInElement`](#waitfortexttobepresentinelement)
+  - [`waitForTextNotToBePresentInElement`](#waitfortextnottobepresentinelement)
+  - [`waitForUrlToBeEqualToExpectedUrl`](#waitforurltobeequaltoexpectedurl)
+  - [`waitForUrlNotToBeEqualToExpectedUrl`](#waitforurlnottobeequaltoexpectedurl)
+  - [`waitForUrlToContainString`](#waitforurltocontainstring)
+  - [`waitForUrlNotToContainString`](#waitforurlnottocontainstring)
+  - [`fillFieldWithTextWhenVisibleAndPressEnter`](#fillfieldwithtextwhenvisibleandpressenter)
+  - [`scrollToElementWhenVisible`](#scrolltoelementwhenvisible)
+
+</p> </details>
+
+- [Using methods that start with 'wait' as test expectations (or test assertions)](#using-methods-that-start-with-wait-as-test-expectations-(or-test-assertions))
```

Comment:
Still wrong link. It should be `#using-methods-that-start-with-wait-as-test-expectations-or-test-assertions`.

Reply:
Commit made because this review: a34ef27